code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -132,6 +132,10 @@
 /datum/chemical_reaction/reagent_explosion/tatp_explosion/proc/UpdateInfo()
 	required_temp = 550 + rand(-49,49)
 
+/datum/chemical_reaction/reagent_explosion/tatp_explosion/on_reaction(datum/reagents/holder, created_volume)
+	strengthdiv = 3 - (created_volume >= 2100 ? 2 : round(created_volume/1400)) //formula that increases the potency of the explosion the more chem is used making tatp a top-tier instead of mid-tier
```

Comment:
shouldn't round(created_volume/1400) be round(created_volume/1050)? The strengthdiv current jumps a bit when you pass the threshold of 2100u

Reply:
i used a graph calculator to determien the value and yeah it would normally jump if not for the round(). which ceils if the value is nearer 1 or floors if it is nearer 0 . 

https://www.desmos.com/calculator/6gawdla4ad < 1050

https://www.desmos.com/calculator/irzc0elnev < 1400

Reply:
oh and the ? expression is just in case someone decides to spawn in more than 2100u of tatp so that we dont divide by 0